Sources/AWSSDKSwiftCore/Encoder/AWSShapeEncoder.swift

```diff
@@ -120,7 +120,8 @@ class AWSShapeEncoder {
                         dictionary["\(fullLabel).\(entryString)\(iterator.offset+1).\(keyString)"] = iterator.element.key
                         dictionary["\(fullLabel).\(entryString)\(iterator.offset+1).\(valueString)"] = iterator.element.value
                     }
-                
+                case let v as Data:
+                    dictionary[fullLabel] = String(decoding: v, as: UTF8.self)
```

Comment:
> I can pretty much guarantee any other service that supplies Data objects in a query string requires them to be base64 encoded as well.

So, just replace this line to `dictionary[fullLabel] = v.base64EncodedString()` and test it?

Reply:
Yep. I believe that is the correct way to do this.

Reply:
I see, I will change it later.

Reply:
@adam-fowler I have modified, please review again.